.github/workflows/linux_release.yml

```diff
@@ -0,0 +1,78 @@
+name: Linux Releases
```

Comment:
This is the actual workflow that builds Linux release packages for us now.  Key points:

- Runs on Ubuntu 18.04 and 20.04
- Is run only on tags, not regular commits
- Builds EnergyPlus against Python 3.7
- Builds and packages up the expected installer types: SH, IFW, and TAR